src/Communication/SAF.Communication.PubSub.Cde/SubscriptionRegistry.cs

```diff
@@ -276,6 +273,13 @@ private void HandleSubscriberAlive(TSM message)
                 {
                     _syncSubscribers.ExitWriteLock();
                 }
+                if (string.IsNullOrWhiteSpace(message.PLS))
+                {
+                    var tsmAlive = new TSM(Engines.PubSub, MessageToken.RegistryAlive, _registryIdentity);
```

Comment:
We should answer to `message.ENG` instead of `Engines.PubSub` to catch the case of "remote subscribers" too.